dissect/target/plugins/apps/browser/chromium.py

```diff
@@ -170,6 +174,46 @@ def downloads(self, browser_name: str = None) -> Iterator[BrowserDownloadRecord]
             except SQLError as e:
                 self.target.log.warning("Error processing history file: %s", db_file, exc_info=e)
 
+    def cookies(self, browser_name: str = None) -> Iterator[BrowserCookieRecord]:
+        """Return browser cookies records from supported Chromium-based browsers.
+        Args:
+            browser_name: The name of the browser as a string.
+        Yields:
+            Records with the following fields:
+                ts_created (datetime): Download start timestamp.
+                ts_last_accessed (datetime): Download end timestamp.
```

Comment:
Is this the correct description? Should this not be related to the creation and last access of the cookie?